src/App.js

```diff
@@ -1,10 +1,23 @@
 import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import 'normalize.css';
+import './components/styles/App.css';
```

Comment:
I would keep the styles next to the component. This is common practice on React and makes maintenance much easier when the so grows since you have everything on one place